src/options/backup.tsx

```diff
@@ -13,21 +13,18 @@ export const Backup = () => {
   const [syning, setSyning] = createSignal(false)
   const [latestSyncTime, setLatestSyncTime] = createSignal(0)
   const [syncFailedMessage, setSyncFailedMessage] = createSignal('')
-  const [authToken, setAuthToken] = createSignal('')
+  const [gdriveToken, setGdriveToken] = createSignal('')
   const [githubSyning, setGithubSyning] = createSignal(false)
-  const [githubToken, setGithubToken] = createSignal('')
-  const [githubGistId, setGithubGistId] = createSignal('')
   const [latestGistSyncTime, setLatestGistSyncTime] = createSignal(0)
   const [gistSyncFailedMessage, setGistSyncFailedMessage] = createSignal('')
 
-  const onAuthTokenInput = (e: Event) => {
+  const onGDriveTokenInput = (e: Event) => {
     const target = e.target as HTMLTextAreaElement
-    setAuthToken(target.value)
     if (target.value.trim() === '') {
-      chrome.storage.local.remove([StorageKey.mobile_auth_token])
+      setSetting('githubToken', '')
     } else {
       if (isValidAuthToken(target.value)) {
-        chrome.storage.local.set({ [StorageKey.mobile_auth_token]: target.value })
+        setSetting('githubToken', target.value)
```

Comment:
Is it correct to set the githubToken in the onGDriveTokenInput method?

Reply:
It's a mistake, resolved now.